File: C++/49_Group_Anagrams.cpp
class Solution {
public:
    vector<vector<string>> groupAnagrams(vector<string>& strs) {
        unordered_map<string, vector<string>> d;
        for (string s : strs){
            string t = s;
            sort(s.begin(), s.end());
            d[s].push_back(t);
        }
        vector<vector<string>> l;
        for (auto x : d) {
            l.push_back(x.second);
        }
        return l;
    }
};

File: Python/49_Group_Anagrams.py
class Solution:
    def groupAnagrams(self, strs: List[str]) -> List[List[str]]:
        d = {}
        for i in range(len(strs)):
            word = ''.join(sorted(strs[i]))
            if word not in d:
                d.setdefault(word, [])
            d[word].append(strs[i])
        l = []
        for i in d.values():
            l.append(i)
        return l